### ISSUE TITLE
Reverse proxy URLs not handling correctly

### ISSUE THREAD

**Issue by kvndl** [YodaDaCoda/chrome-deluge-remote#23]
Since I like to minimize the amount of ports I have open on my server, I use reverse proxy for everything I access (domain.com/deluge). Unless I am doing something wrong here, I cannot connect when not using a port number. I'm sure if I opened the port and used that instead it would work fine, but just wondering if this type of functionality is possible.

![image](https://user-images.githubusercontent.com/22201748/63982053-6c712800-ca8f-11e9-8468-5267ce9ecfd6.png)

**Comment by Xe138** [comment]
I'm also using a reverse proxy for this.  Instead of using the deluge port, it should work if you use the port number that your proxy is configured to listen on.

**Comment by kvndl** [comment]
I have opened the deluge port, just in case that may have been causing issues and set it up the way you have described as well. [Here is my current setup using Nginx](https://i.imgur.com/4nbOwqu.png), and [here is how I set it up in the extension options](https://i.imgur.com/vYFLS83.png). I can access it normally by going to https://domain.com/deluge with my current setup, but I'm not for sure if the extension requires a port number to be passed through it. I currently use delugesiphon, which just requires a raw URL to be passed through it, instead of individual fields for a port and handle.

I get the same error when trying it with the port number included: "Unable to connect.."
